labels-annotation-for-openshift.adoc

```diff
@@ -57,6 +57,16 @@ TIP: Labels must be applied to ALL resources, DeploymentConfigs, BuildConfigs, S
 | OPT
 | The current version of the component (e.g., a semantic version, revision hash, etc.)
 | 1.0.0
+
+| app.openshift.io/runtime
+| OPT
+| The runtime for used for running the component
```

Comment:
I propose to rename the phrase to `The runtime to be used to bootstrap the component`. Make sense ?